src/pages/Home/index.jsx

```diff
@@ -173,23 +173,30 @@ const Home = ({ location }) => {
                       />
                     </Col>
                   </Form.Group>
-                  <Button
-                    type="submit"
-                    onClick={handleSubmit}
-                    className="d-block mx-auto"
-                  >
-                    {loading ? (
-                      <Spinner
-                        as="span"
-                        animation="border"
-                        size="sm"
-                        role="status"
-                        aria-hidden="true"
-                      />
-                    ) : (
-                      "Login"
-                    )}
-                  </Button>
+                  <Form.Group as={Row}>
+                    <Form.Label column sm="4">
+                      &nbsp;
+                    </Form.Label>
```

Comment:
Is this `Label` needed here?

Reply:
I think that is causing the button to go down in mobile view.

Reply:
Okay I am looking into it

Reply:
If I remove the mentioned block  this is the case

Desktop/Laptop view:
![image](https://user-images.githubusercontent.com/63705023/155069060-6fe75bd6-4329-48cf-b0aa-89274fb046d8.png)


Mobile View:
![image](https://user-images.githubusercontent.com/63705023/155069119-16030c48-283a-44cb-ba46-e47499f3f945.png)


Do let me know if this is acceptable or we need further changes
@GMishx

Reply:
@GMishx  Please do have a look 

Mobile view: 
![image](https://user-images.githubusercontent.com/63705023/155073127-69b4a20c-f445-4225-9c48-9708a3d2e842.png)


Desktop view:
![image](https://user-images.githubusercontent.com/63705023/155073193-28a4946b-7f7c-4366-bf8b-e811e13d407a.png)

Reply:
Looks better now :+1: